.github/workflows/test.yaml

```diff
@@ -0,0 +1,43 @@
+name: Test Charts
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2
```

Comment:
Please see Apache [policy](https://infra.apache.org/github-actions-policy.html) on using third-party actions.  We need to pin such actions by SHA, instead of using symbolic version names, which are movable by the third-party.

Checking a few similar projects, they seem to do so by linking to those specific versions in `.github/actions`, e.g. https://github.com/apache/superset/tree/master/.github/actions

Reply:
@adoroszlai Thanks. I didn't know about the policy. If I'm not mistaken, the following also fixes the issue: 
```
      - name: Set up chart-testing
        uses: helm/chart-testing-action@e6669bcd63d7cb57cb4380c33043eebe5d111992 # v2.6.1
```
Am I right?

Reply:
Yes, this is also good.  I guess the other approach is preferable if the same action is used multiple times, because it reduces change required to upgrade.